app/assets/stylesheets/components-vue/_social.scss

```diff
@@ -1,92 +1,118 @@
+// Height of popup share arrow
+$popup-share-arrow-height: rem-calc(11);
 //--------------------------------------------------
-// social
+// mixins
+//--------------------------------------------------
+@mixin social-basic ($image) {
+  background-image: image-url($image);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: contain;
+
+  display: inline-block;
+
+  &:hover { opacity: .75; } 
+}
+
+@mixin social-icon ($image) {
+  @include social-basic($image);
+
+  @include breakpoint($small) {
+    margin-right: rem-calc(14);
+    width: rem-calc(41); 
+    height: rem-calc(41);
+  }
+}
+
+@mixin social-icon-white ($image) {
+  @include social-basic($image);
+  width: rem-calc(41); height: rem-calc(41);
+
+  &:first-child { 
+    margin-top: rem-calc(10);
+  }
+
+  @include breakpoint($small) {
+    &:first-child {
+      margin-top: 0;
+    }
+  }
+}
+
+//--------------------------------------------------
+// classes
 //--------------------------------------------------
 .social {
 //--------------------------------------------------
-// social
+// following or sharing via social media 
 //--------------------------------------------------
-  &--share {
+  &--media {
+    @include flex;
     position: relative;
 
-    .social {
-      &__trigger {
-        margin: rem-calc(0 20 0 17);
+    .social__icon {
+      &--email { @include social-icon('icons/email.svg'); }
+      &--facebook {
+        @include social-icon-white('icons/facebook-white.svg');
 
-        display: block;
-        
-        @include breakpoint($small) { margin: rem-calc(0 30 0 22); }
-        @include breakpoint($medium) { display: none; }
+        @include breakpoint($small) {
+          @include social-icon('icons/facebook.svg'); 
+        } 
       }
+      &--linkedin { 
+        @include social-icon-white('icons/linkedin-white.svg');
 
-      &__target {
```

Comment:
noticed you've removed the medium breakpoint styles - are we only going with mobile and desktop?